web-components/src/components/breadcrumbs/Breadcrumbs.ts

```diff
@@ -0,0 +1,78 @@
+import reset from "@/wc_scss/reset.scss";
+import { customElement, html, LitElement, property, PropertyValues, queryAll } from "lit-element";
+import styles from "./scss/module.scss";
+import { repeat } from "lit-html/directives/repeat";
+import { ifDefined } from "lit-html/directives/if-defined";
+
+type NavCrumb = {
+  link: string;
+  label: string;
+};
+
+@customElement("md-breadcrumbs")
+export class Breadcrumbs extends LitElement {
+  @property({ type: String }) label = "";
```

Comment:
according to the wai-aria best practice this should default to "Breadcrumb"

Reply:
fixed

Reply:
Did it get changed back? I still see an empty string.

Reply:
Done